integrations/acquisition/covid_hosp/state_timeseries/test_scenarios.py

```diff
@@ -35,7 +38,7 @@ def setUp(self):
     secrets.db.epi = ('user', 'pass')
 
     # clear relevant tables
-    with Database.connect() as db:
+    with Database().connect() as db:
       with db.new_cursor() as cur:
         cur.execute('truncate table covid_hosp_state_daily')
```

Comment:
Removing those two lines causes a test failure in `state_timeseries` - I believe this would be fixed once the other PR related to those tables https://github.com/cmu-delphi/delphi-epidata/pull/1126 is merged.

Reply:
ah, ok, good call.  how about we leave a TODO here that reminds us to remove the `state_daily` when #1126 is merged in?